android/app/src/main/java/org/jetbrains/kotlinconf/KotlinConfApplication.kt

```diff
@@ -1,16 +1,21 @@
 package org.jetbrains.kotlinconf
 
-import android.app.*
-import android.content.*
-import android.support.multidex.*
-import kotlinx.coroutines.*
-import kotlinx.coroutines.android.*
-import org.jetbrains.anko.*
-import org.jetbrains.kotlinconf.model.*
-import java.util.*
+import android.app.Application
```

Comment:
Could you use `*` star imports everywhere instead?

Reply:
Sure